logdna/configs.py

```diff
@@ -3,5 +3,6 @@
     'MAX_LINE_LENGTH': 32000,
     'FLUSH_INTERVAL': 5,
     'FLUSH_BYTE_LIMIT': 2 * 1024 * 1024,
-    'LOGDNA_URL': 'https://logs.logdna.com/logs/ingest'
+    'LOGDNA_URL': 'https://logs.logdna.com/logs/ingest',
+    'BUFFER_BYTE_LIMIT': 4194304
```

Comment:
why `4mb`?

Reply:
it is it flush limit * 4.  I did not really put a lot of thought into this number. Can be changed.